system_tests/bigtable.py

```diff
@@ -384,3 +390,95 @@ def test_read_row(self):
             },
         }
         self.assertEqual(partial_row_data.cells, expected_row_contents)
+
+    def test_read_rows(self):
+        row = self._table.row(ROW_KEY)
+        row_alt = self._table.row(ROW_KEY_ALT)
+        self.rows_to_delete.extend([row, row_alt])
+
+        cell1, cell2, cell3, cell4 = self._write_to_row(row, row_alt,
+                                                        row, row_alt)
+        row.commit()
+        row_alt.commit()
+
+        rows_data = self._table.read_rows()
+        self.assertEqual(rows_data.rows, {})
+        rows_data.consume_all()
+
+        # NOTE: We should refrain from editing protected data on instances.
```

Comment:
The joys of copying and pasting code (even though I wrote it). I'll file an issue but agree that it should be tackled. I'm kind of at a loss for how to tackle this, other than providing a factory. WDYT?